projects/packages/image-cdn/src/class-image-cdn.php

```diff
@@ -279,9 +279,40 @@ public static function filter_photon_norezise_maybe_inject_sizes_api( $sizes, $a
 	 * * IN-CONTENT IMAGE MANIPULATION FUNCTIONS
 	 **/
 
+	/**
+	 * Given a `class` attribute value, return the first class name with
+	 * the given prefix, excluding that prefix in the return value.
+	 *
+	 * @since {VERSION NUMBER}
```

Comment:
We use `$$next-version$$` as the placeholder:

https://github.com/Automattic/jetpack/blob/3a3c4adc1c0913bac224ddb832713bd6101fe1ab/projects/packages/README.md#L107-L116

```suggestion
	 * @since $$next-version$$
```

Reply:
nice, thank you @jeherve!